### DISCUSSION TITLE
Rotating the mesh

### DISCUSSION THREAD

**Discussion by kasia129** [capytaine/capytaine#194] [Q&A]
Edit: the below has been solved by reinstalling Capytaine. The bug must have been fixed lately.

Hi, I am trying to rotate a mesh around a given axis. My workflow is as follows:

1) Create the basis (not inclined) mesh: 

```
AxialSymmetricMesh = cpt.AxialSymmetricMesh.from_profile(profile=shape, axis=cpt.meshes.geometry.Axis(vector=[0., 0., 1.], point=[0., 0., 0.]), nphi=20)
buoy = cpt.FloatingBody(AxialSymmetricMesh)
```

3) rotate the basis mesh around the given axis:

```
my_axis = Axis(vector=[0, 1, 0], point=[0, 0, -60])
rotated_buoy = buoy.rotated(axis=my_axis, angle=10/180*np.pi)
```

4) plot the two meshes in one plot:

```
both = cpt.FloatingBody.join_bodies(buoy, rotated_buoy, name='both')
both.show_matplotlib() 
```

**Problem**: It looks like the mesh is being rotated around the axis located at the 0,0,0 point, rather than 0,0,-60. Do you have any idea why that may be? Please see the plot below.

![image](https://user-images.githubusercontent.com/76471057/184623498-04bdb198-76ea-4631-88fe-3b014d5bc909.png)

A reproducible example is included below FYI.

I would be grateful for any hints.

```
import numpy as np
import capytaine as cpt
import matplotlib.pyplot as plt
from capytaine import Axis

# input
r_vec = [5,  5,  5,  5]
T = 120

# profile shape for axisym. mesh
dT = T/(len(r_vec)-1)
shape = []
for i,r in enumerate(r_vec):  
    shape.append([r,  0,  -i*dT])
    
# generate the upright mesh    
AxialSymmetricMesh = cpt.AxialSymmetricMesh.from_profile(profile=shape, axis=cpt.meshes.geometry.Axis(vector=[0., 0., 1.], point=[0., 0., 0.]), nphi=20)
buoy = cpt.FloatingBody(AxialSymmetricMesh)

# rotate the mesh    
my_axis = Axis(vector=[0, 1, 0], point=[0, 0, -60])
rotated_buoy = buoy.rotated(axis=my_axis, angle=10/180*np.pi)

# plot both floaters
both = cpt.FloatingBody.join_bodies(buoy, rotated_buoy, name='both')
both.show_matplotlib() 


```

**Comment by mancellin** [comment]
> Edit: the below has been solved by reinstalling Capytaine. The bug must have been fixed lately.

Indeed, it looks like #151 which has been fixed in version 1.4 of Capytaine.
